opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -676,7 +676,12 @@ def __exit__(
 
         super().__exit__(exc_type, exc_val, exc_tb)
 
-    def record_exception(self, exception: Exception) -> None:
+    def record_exception(
+        self,
+        exception: Exception,
+        attributes: types.Attributes = None,
+        timestamp: Optional[int] = None,
```

Comment:
Is `timestamp` defined in the specs?

Reply:
From specs

> To facilitate recording an exception languages SHOULD provide a RecordException method if the language uses exceptions. This is a specialized variant of AddEvent, so for anything not specified here, the same requirements as for AddEvent apply

> An API to record a single Event where the Event properties are passed as arguments. This MAY be called AddEvent. This API takes the name of the event, optional Attributes and an optional Timestamp which can be used to specify the time at which the event occurred